components/raftstore-v2/src/operation/command/admin/split.rs

```diff
@@ -148,6 +149,7 @@ pub struct RequestSplit {
     pub epoch: RegionEpoch,
     pub split_keys: Vec<Vec<u8>>,
     pub source: Cow<'static, str>,
+    pub amortize_source_region_size: bool,
```

Comment:
Please add some comments to describe it.

Reply:
Including which cases it will be true/false.

Reply:
fixed